.github/workflows/integration_tests.yml

```diff
@@ -58,7 +64,13 @@ jobs:
         run: |
           sbt integrationTests/test
 
-      # TODO Upload metrics and logs to S3
+      - name: Save logs
+        run: |
+          mkdir -p /tmp/upload
+          while read ip; do
+            scp -o StrictHostKeyChecking=no admin@$ip:/home/admin/constellation/logs/app.log "/tmp/upload/app-${ip}.log"
+          done < terraform/aws/hosts-integration-tests-$EXECUTION_TAG
+          aws s3 cp /tmp/upload/ s3://constellationlabs-dag/logs/$EXECUTION_TAG --recursive
```

Comment:
there are also archived logs - it depends how long the cluster is running thus how much logs it generated.
also it would be great to prefix the logs on s3 by the execution number.

Reply:
For now log rotation is set to some defaults but it can be configurable via:

```xml
  <rollingPolicy class="LogRotate">
      <periodMinutes>5</periodMinutes>
      <maxHistory>30</maxHistory> // max days
      <totalSizeCap>3GB</totalSizeCap> // max size
    </rollingPolicy>
```

So in theory we can modify that  depending on test needs

Reply:
@marcinwadon I will add archived logs as well, thanks for catching that. The logs are in the s3 directory `logs/$EXECUTION_TAG` with the IP of the node as a file suffix. But I will change that to `logs/$EXECUTION_TAG/$ip/`

@kpudlik I think I will keep the default config, if there will be very long-running tests, compression and log rotation may be useful.

Reply:
What is the execution tag? Is it a unique tag per execution or terraform workspace?

Reply:
Both. It's a suffix added to terraform workspace name, and it should be unique to the execution.